src/utils/index.ts

```diff
@@ -72,6 +72,7 @@ export async function depositKodiak(
   island: AddressArg,
   primary: AddressArg,
   setter: AddressArg,
```

Comment:
This function doesn't need to be passed the setter address. You can look it up using the `builder.chainId` value